### PR TITLE
docs: Add Configuration tutorial

### PR DESCRIPTION
**Pull request by lukeblevins**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by carldebilly — APPROVED** [review]

File: doc/Learn/Tutorials/Configuration/HowTo-Configuration.md [hunk 1]
```diff
@@ -1,10 +1,95 @@
...
+            .CreateDefaultBuilder()
+            .UseConfiguration(configure: configBuilder =>
+                configBuilder
+                    // Load configuration information from appsettings.json
+                    .EmbeddedSource<App>()
+            )
+    ...
+    ```
+
+* By default, this method will extract values from an embedded resource called `appsettings.json`, unless you optionally denote a different file name. The string you pass into the extension method will be concatenated in between `appsettings` and its file extension. For instance, the following will also retrieve values from the file `appsettings.platform.json` embedded inside the `App` assembly:
```
Comment (jeromelaban) [inline doc/Learn/Tutorials/Configuration/HowTo-Configuration.md:29]:
```suggestion
* By default, this method will extract values from an embedded resource (using the `EmbeddedResource` file build action) called `appsettings.json`, unless you optionally denote a different file name. The string you pass into the extension method will be concatenated in between `appsettings` and its file extension. For instance, the following will also retrieve values from the file `appsettings.platform.json` embedded inside the `App` assembly:
```

File: doc/Learn/Tutorials/Configuration/HowTo-Configuration.md [hunk 2]
```diff
@@ -1,10 +1,95 @@
 ---
 uid: Learn.Tutorials.Configuration.HowToConfiguration
 ---
-# How-To: Configuration
+# How-To: Get Started with Configuration
 
-Configuration in Uno.Extensions provides you with a consistent way to view configuration data from a number of distinct sources. This pattern is not intended to allow for programmatic writes by itself, but instead offers a single representation of any number of providers like `appsettings.json`.
+Configuration in Uno.Extensions provides you with a consistent way to view configuration data from a number of distinct sources. This pattern is not intended to allow for programmatic writes by itself, but instead offers a single representation of any number of providers or sections.
```
Comment (jeromelaban) [inline doc/Learn/Tutorials/Configuration/HowTo-Configuration.md:6]: We probably should be explaining what a "section" is.

**Review by jeromelaban — APPROVED** [review]

File: doc/Learn/Tutorials/Configuration/HowTo-Configuration.md (hunk 2 above)
Comment (agneszitte-nventive) [inline doc/Learn/Tutorials/Configuration/HowTo-Configuration.md:6]:
```suggestion
Configuration in Uno.Extensions provide you with a consistent way to view configuration data from a number of distinct sources. This pattern is not intended to allow for programmatic writing by itself, but instead offers a single representation of any number of providers or sections.
```

File: doc/Learn/Tutorials/Configuration/HowTo-Configuration.md (hunk 1 above)
Comment (agneszitte-nventive) [inline doc/Learn/Tutorials/Configuration/HowTo-Configuration.md:29]:
```suggestion
* By default, this method will extract values from an embedded resource called `appsettings.json`, unless you optionally denote a different file name. The string you pass into the extension method will be concatenated in-between `appsettings` and its file extension. For instance, the following will also retrieve values from the file `appsettings.platform.json` embedded inside the `App` assembly:
```

**Pull request merged**